components/payments/cmd/connectors/internal/connectors/atlar/task_fetch_payments.go

```diff
@@ -169,12 +169,17 @@ func determinePaymentStatus(item *atlar_models.Transaction) models.PaymentStatus
 		// which was not yet reconciled with a payment from the statement
 		return models.PaymentStatusPending
 	}
-	if item.Reconciliation.Status == atlar_models.ReconciliationDetailsStatusRECONCILED {
+	if item.Reconciliation.Status == atlar_models.ReconciliationDetailsStatusBOOKED {
+		// A payment comissioned with the bank, which was not yet reconciled with a
+		// payment from the statement
 		return models.PaymentStatusPending
 	}
-	if item.Reconciliation.Status == atlar_models.ReconciliationDetailsStatusBOOKED {
+	if item.Reconciliation.Status == atlar_models.ReconciliationDetailsStatusRECONCILED {
 		return models.PaymentStatusSucceeded
 	}
+	if item.Reconciliation.Status == "RETURNED" { // atlar_models is missing an enum value
```

Comment:
This value is also not documented, but they told me it does exist.

Reply:
Apparently there was some confusion about payments and transactions. The mapping works as follows

- Formance `Payment` -> Atlar `Transaction`
- Formance `Payment Initiation` -> Atlar `Payment`

The payment object on one platform does not equal the payment object on the other platform. That led to some confusion. This PR still fixes something I just understood. I will fix updating `Payment Initiation` status in another PR.